src/js/experiments/languageGuides.js

```diff
@@ -0,0 +1,81 @@
+const displayElement = (el, display) => {
+  if (el) {
+    el.style.display = display;
+  }
+};
+
+const displayInitialElement = (el) => displayElement(el, 'initial');
+const displayBlockElement = (el) => displayElement(el, 'block');
+
+// https://app.optimizely.com/v2/projects/16812830475/experiments/20872380274/variations
+export default () =>
+  window.OptimizelyClient.getVariationName({
+    experimentKey: 'dd_add_video_tab_to_docs_test',
+    groupExperimentName: 'q4_fy22_docs_disco_experiment_group_test',
+    experimentContainer: '.treatment',
+    attributes: {
+      // https://app.optimizely.com/v2/projects/16812830475/audiences/20680670160
+      windowWidth: window.innerWidth ?? 0,
+    },
```

Comment:
Are we preventing people on mobile to see that experiment?

Reply:
not intentional. i'll remove it.